content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -208,29 +129,3 @@ problems:
 
     If you do not see a login prompt and you get a 404 error, the configuration
     of Cloud IAP is not yet complete. Keep retrying for up to 10 minutes.
-
-1. If you get an error `Error: redirect_uri_mismatch` after logging in, this 
```

Comment:
Isn't this still relevent?

Reply:
We don't need per domain redirect uri now right? I check my client, it has only https://iap.googleapis.com/v1alpha1/oauth/clientIds/${ClientID}.apps.googleusercontent.com:handleRedirect

Reply:
I think you could still get a redirect_uri_mismatch error I think it would just happen now because the ClientID was wrong.

Reply:
@lluunn should we restore the docs about handling uri_mismatch?